src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -1030,13 +1024,7 @@ private void AggregateRenderPackages(PackageAggregationParams parameters)
 
                     var endIdx = lineSet.Positions.Count;
 
-                    if (rp.IsSelected)
-                    {
-                        for (var i = startIdx; i < endIdx; i++)
-                        {
-                            lineSet.Colors[i] = selectionColor;
-                        }
-                    }
+                    lineGeometry3D.IsSelected = rp.IsSelected;
 
                     if (rp.DisplayLabels)
```

Comment:
I think IsSelected can be moved to the constructor? 
pointGeometry3D = new PointGeometryModel3D
                        {
                            Geometry = HelixRenderPackage.InitPointGeometry(),
                            Transform = Model1Transform,
                            Color = SharpDX.Color.White,
                            Figure = PointGeometryModel3D.PointFigure.Ellipse,
                            Size = DefaultPointSize,
                            IsHitTestVisible = true,
                            IsSelected = rp.IsSelected
                        };